### DISCUSSION TITLE
Integration with a dropshipping service

### DISCUSSION THREAD

**Discussion by simonwhitaker** [saleor/saleor#8423] [Ideas]
Do you have any plans to integrate with dropshipping services such as [Dropified](https://www.dropified.com/)? 

I'm currently choosing a new eCommerce platform and as a developer I'd love to use Saleor, but we may need dropshipping functionality and that's pushing us towards alternatives like WooCommerce or Shopify. Is there any chance we can have our cake and eat it too? Saleor with integration with Dropified (or similar) would be great!

**Comment by karolkielecki** [comment]
@simonwhitaker Hi, there is no plan to integrate Saleor with dropshipping services. If you're looking for a marketplace integration there is one with Jetti: https://jetti.io/integrations/store/saleor

**Comment by KimolO** [comment]
@simonwhitaker, I did the integration of Saleor as a 3rd party app at https://amphoralogistics.com/ outsourcing fulfillment
The key here would be to **promote** 3rd party apps, and improve the installment process (currently is still necessary a developer to execute a python command)

**Comment by mapmychannel** [comment]
That’s a really good point. Dropshipping integrations like Dropified could definitely make Saleor more attractive to merchants who don’t want to manage inventory. While Shopify and WooCommerce already have these options built in, having a similar integration in Saleor would give developers more flexibility without sacrificing performance or customization. Even if it’s not a core feature, an app or plugin approach might work well so the community can extend it. I’d also be interested to know if Saleor has any roadmap plans for dropshipping support.
